packages/components/src/Disclosure/Disclosure.css

```diff
@@ -9,13 +9,17 @@
 }
 
 .summary {
-  padding: var(--space-smaller) 0 var(--space-smallest) 0;
+  --disclosure--interactionOffset: var(--space-smaller);
+  margin-left: calc(-1 * var(--disclosure--interactionOffset));
+  padding: var(--space-smaller) 0 var(--space-smallest)
+    var(--disclosure--interactionOffset);
   border: none;
+  border-radius: var(--radius-small);
   list-style: none;
   cursor: pointer;
-  outline-color: none;
+  outline-color: transparent;
   transition:
-    background var(--timing-quick),
+    background-color var(--timing-base) ease-out,
```

Comment:
`background` applies to (4?) multiple `background-` properties, `background-color` is specific to the one property we actually need here, and it works, where prior it did not.